services/slam/fake/slam_test.go

```diff
@@ -43,10 +43,13 @@ func TestFakeSLAMGetPosition(t *testing.T) {
 }
 
 func TestFakeSLAMStateful(t *testing.T) {
-	t.Skip("Skipping this test because it takes too long. Re-enable in RSDK-3341")
 	t.Run("Test getting a PCD map via streaming APIs advances the test data", func(t *testing.T) {
+		orgMaxDataCount := maxDataCount
+		// maxDataCount lowered under test to reduce test runtime
+		maxDataCount = 5
```

Comment:
[opt] pass in a maxDataCount to `verifyGetPointCloudMapStateful` rather than modifying a global variable